components/oneapi/src/main/java/com/wso2telco/dep/oneapi/constant/payment/PaymentKeyConstants.java

```diff
@@ -0,0 +1,14 @@
+package com.wso2telco.dep.oneapi.constant.payment;
```

Comment:
Add the license header.

Reply:
/**
 * Copyright (c) 2019, APIGATE PVT (Ltd). (https://www.apigate.com/) All Rights Reserved.
 *
 * APIGATE Inc. licences this file to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 * http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */

Reply:
Done